tests/unit/customizations/s3/test_subcommands.py

```diff
@@ -400,6 +400,48 @@ def test_validate_no_streaming_paths(self):
         cmd_params.add_paths(paths)
         self.assertFalse(cmd_params.parameters['is_stream'])
 
+    def test_validate_checksum_algorithm_download_error(self):
+        paths = ['s3://bucket/key', self.file_creator.rootdir]
+        parameters = {'checksum_algorithm': 'CRC32'}
+        cmd_params = CommandParameters('cp', parameters, '')
+        with self.assertRaises(ParamValidationError):
```

Comment:
`ParamValidationError` is used enough throughout the codebase that when testing we should assert that the error message is what we expect (and not propagating some other `ParamValidationError`).

Reply:
Good point, resolving in next revision